src/NuGet.Services.AzureSearch/Analysis/TruncateCustomTokenFilter.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Search.Models;
+
+namespace NuGet.Services.AzureSearch
+{
+    /// <summary>
+    /// Truncates tokens to 300 characters or less. This is necessary as Azure Search's
+    /// <see cref="TruncateTokenFilter"/> defaults to 10 characters.
+    /// </summary>
+    public static class TruncateCustomTokenFilter
+    {
+        public const string Name = "nuget_truncate_filter";
```

Comment:
We can add this to the integration test suite, right?

Reply:
Yes, in fact, I believe the existing integration tests break because of this change. Thanks for the reminder :)

Reply:
Added